python/base/Dockerfile

```diff
@@ -0,0 +1,6 @@
+# Configure a Debian testing distro to run Tox for Python 2.7 and 3.4
+FROM ubuntu:trusty
+MAINTAINER José Armando García Sancio <jose@mesosphere.io>
+RUN apt-get update && \
+    apt-get install -y python2.7 python3.4 python-pip python-dev && \
```

Comment:
I think we should do the following: `apt-get install -y python python3 python-pip python3-pip python-dev python3-dev`